app/presenters/vendor_api/application_presenter/add_inactive_status.rb

```diff
@@ -0,0 +1,11 @@
+module VendorAPI::ApplicationPresenter::AddInactiveStatus
+  def schema
+    return super unless @application_choice.inactive?
+
+    super.deep_merge(
+      attributes: {
+        status: 'inactive',
```

Comment:
Unfortunately adding the "inactive" status will be classed as a breaking change as this will have a negative impact on API integrations. 

Could we change this to something like 
```suggestion
        inactive: @application_choice.inactive?,
```
Where the `inactive` attribute becomes a boolean value?

Reply:
This is not a breaking change. In version `1.6` (the version being introduced here), the status field will return inactive when applicable. For all other API versions, the status field will continue to return `awaiting_provider_decision`, ensuring backward compatibility.

Introducing a new field to represent a status that’s already part of the status enum feels unnecessary and could lead to confusion for API consumers. It adds complexity, as clients would need to check multiple fields to understand the application status.

Keeping status as the single source of truth is cleaner, more consistent, and aligns with the existing design.

As an extra precaution, I’ll also add a test for version `1.5` to ensure it continues to return `awaiting_provider_decision`.

Reply:
I’ve added tests to all previous versions to maintain coverage and ensure the API continues to return `awaiting_provider_decision` for all versions prior to `1.6`.

https://github.com/DFE-Digital/apply-for-teacher-training/pull/10189/commits/678ce46a7540464408dcd82fd86a4512a6c8cf3a

Reply:
Had a chat offline and we’ve decided to implement this as a boolean field, as vendors have been assured it would be. 

This also highlighted the need to eventually remove `inactive` from the `ApplicationChoice` statuses as part of tech debt work.